setupdocs.md

```diff
@@ -0,0 +1,182 @@
+
+# Car-Bon System: Technical Setup Guide
+
+## Prerequisites
+
+Before beginning the installation, ensure you have:
+
+-   Administrator access to the project repository of github
+-   Windows operating system
+-   Basic understanding of command-line interfaces
+
+## Version Control Setup
+
+1.  Install Git
+    -   Download and install from the official website
+    -   Configure with your credentials
+2.  Clone Project Repository
+    
+    bash
+    
+    Copy
+    
+    `git clone https://github.com/gunnnu/Car-Bon_UI.git`
+    
+    **Note:** Repository access requires admin invitation
+
+## Windows Subsystem for Linux (WSL) Configuration
+
+1.  Open PowerShell as Administrator
+    
+    powershell
+    
+    Copy
+    
+    `wsl --install`
+    
+2.  Complete Ubuntu installation and create user account
+
+## Python Environment Preparation
+
+1.  Install Python 3.9+
+    
+    bash
+    
+    Copy
+    
+    `python3 --version`
+    
+2.  Configure Python System
+    
+    bash
+    
+    Copy
+    
+    `sudo  apt  install python-is-python3 sudo update-alternatives --install /usr/bin/python python $(readlink -f $(which python3))  3`

```

Comment:
1. The install command should not be here , it should be in setup section
2. Refrain from using AI tools for documentation refer above comments for how to correctly write commands in docs
3. This is wrong command as I mentioned yesterday as well. You have to use pyenv to setup python as that is what we used in the actual system.